feg/gateway/tools/swx_cli/main.go

```diff
@@ -235,6 +235,10 @@ func sendSar(addr string, client swxClient) int {
 }
 
 func sendMar(addr string, client swxClient) int {
+	if numVectors < 0 || numVectors > 4294967295 {
```

Comment:
So for some reason this PR did not go through the feg-workflow. Any ideas why not e.g. @quentinDERORY?

feg-workflow linting fails on this line because `numVectors` is a unsigned integer and cannot be less than zero.

Presently in Master, this blocks all PRs that execute feg-workflow (a required workflow when run).  See [feg-workflow failure of my PR](https://github.com/magma/magma/pull/10415).  I'll send out a PR now to modify this and make master healthy.  No fault of yours @spikey979 - I don't know why the feg-worfklow did not run (I would not expect contributors, internal or other, to know all the tests that need to run for a PR - that is what CI is for).

Reply:
I fixed this in PR: https://github.com/magma/magma/pull/10372

Reply:
I've also posted a fix as of minutes ago - https://github.com/magma/magma/pull/10416 - happy to take whichever, though I have an additional minor nit fixup in my PR (using Golang builtin to detect out of range). Mind if we land mine? Thanks for the prompt fixup PR - we probably should have made this visible / fast tracked it since it broke master but I think nobody knew.

Reply:
Still wondering why the `feg-worfklow` did not run on this PR - which let us get into a bad-master state. Hopefully DevOps folks have an idea there (e.g. @tmdzk or @quentinDERORY )

Reply:
It did run, but the tests failed so early it did not outputed xml outputs. And that's why the comment job is showing no tests runs.
When merging jobs recently I introduced a bug where steps were marked as successful when they were not. (This is why master went into this bad state for feg workflow )
I just resolved it yesterday and now the status of the steps will be displayed correctly.
For the tests not run, we need to improve the comment job to display this specific corner case.

Reply:
created issue for feg tests showing not being run: https://github.com/magma/magma/issues/10421